src/main/src/test/java/org/geoserver/ModuleStatusTest.java

```diff
@@ -0,0 +1,21 @@
+package org.geoserver;
```

Comment:
Unit tests should really have copyright headers too.  W aren't always as picky about them as about implementation files but really they should be everywhere.